Resources/doc/custom-id-generators.rts

```diff
@@ -0,0 +1,64 @@
+Custom ID Generators
+====================
+
+Custom ID generators are classes that allow implementing custom logic to generate
+identifiers for your entities. They extend ``Doctrine\ORM\Id\AbstractIdGenerator``
+and implement the custom logic in the ``generate(EntityManager $em, $entity)``
+method. Before Doctrine bundle 2.3, custom ID generators were always created
+without any constructor arguments.
+
+Starting with Doctrine bundle 2.3, the ``CustomIdGenerator`` annotation can be
+used to reference any services tagged with the ``doctrine.id_generator`` tag.
+
+.. code-block:: php
+
+    <?php
+    // User.php
+
+    use Doctrine\ORM\Mapping as ORM;
+
+    /**
+     * @ORM\Entity
+     */
+    class User
+    {
+        /**
+         * @Id
+         * @Column(type="integer")
+         * @ORM\GeneratedValue(strategy="CUSTOM")
+         * @ORM\CustomIdGenerator('my_id_generator')
```

Comment:
I may be missing something, but `my_id_generator` does not match the config below, which uses the FQCN as identifier for the service.
```suggestion
         * @ORM\CustomIdGenerator(MyIdGenerator::class)
```

Reply:
I was updating the doc while you were reviewing :)